source/routing/raptor.cpp

```diff
@@ -438,14 +440,17 @@ RAPTOR::Journeys RAPTOR::compute_all_journeys(const map_stop_point_duration& dep
     auto starting_points = make_starting_points_snd_phase(*this, calc_dest, accessibilite_params, clockwise);
     swap(labels, first_pass_labels);
 
+    const auto inrows_labels = best_labels.inrow_labels();
+
     auto best_labels_for_snd_pass = Labels(
```

Comment:
Here you create  `inrows_labels`, a full copy  of the data present in `best_labels`, and then you create another copy of this data in `best_labels_for_snd_pass`.
Before, the temporary copy with `inrows_labels` was not needed. 
Could you avoid creating it ?

Reply:
Good catch, Label is now taking `const ref` Map as contructor parameter to avoid the copy 👍🏻